src/helper.js

```diff
@@ -205,6 +207,10 @@ function passwordCompare(password, expected) {
 	return bcrypt.compare(password, expected);
 }
 
+function getHumanDate() {
+	return moment().format("YYYY-MM-DD HH:mm:ss");
```

Comment:
Any reason why our own format instead of [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601)?
I realize it breaks from current default format, but I think it would be nicer/more standard for logs.

Reply:
Our format is effectively 8601 minus the `T` separator, which is also agreed upon, for example in RFC 3339: https://tools.ietf.org/html/rfc3339#section-5.6

      NOTE: ISO 8601 defines date and time separated by "T".
      Applications using this syntax may choose, for the sake of
      readability, to specify a full-date and full-time separated by
      (say) a space character.

Reply:
Fair enough.

Reply:
What about returning `return moment().format()` and letting Moment's default (ISO 8601) do the job?

Reply:
I think having a space is more readable.